File: submitted_code/100280.ti-huan-kong-ge-lcof.cpp
//
// @lc app=leetcode.cn id=100280 lang=cpp
//
// [100280] ti-huan-kong-ge-lcof
//
class Solution {
public:
    string replaceSpace(string s) {
        string res;
        for(int i=0;i<s.size(); ++i){
            if(s[i] == ' ')
                res += "%20";
            else
                res += s[i];
        }
        return res;
    }
};
// @lc code=end

File: submitted code/100280.ti-huan-kong-ge-lcof.java
//
// @lc app=leetcode.cn id=100280 lang=java
//
// [100280] ti-huan-kong-ge-lcof
//
class Solution {
    public String replaceSpace(String s) {
        StringBuilder sb = new StringBuilder();
        for(int i=0;i<s.length();++i){
            if(s.charAt(i) == ' ')
                sb.append("%20");
            else
                sb.append(s.charAt(i));
        }
        return sb.toString();
    }
}
// @lc code=end